logic/src/commonMain/kotlin/com/wire/kalium/logic/data/conversation/MLSConversationRepository.kt

```diff
@@ -173,6 +175,8 @@ internal class MLSConversationDataSource(
     private val epochsFlow: MutableSharedFlow<GroupID>,
     private val proposalTimersFlow: MutableSharedFlow<ProposalTimer>,
     private val keyPackageLimitsProvider: KeyPackageLimitsProvider,
+    private val checkRevocationList: CheckRevocationListUseCase,
```

Comment:
@yamilmedina Maybe we should revisit the architecture tests? It seems this was skipped?

Just mentioning you in case I forget this on Monday xD

I guess some of the functions in this repository shouldn't be here, but on their dedicated classes instead.

Reply:
Now I know. There was added an `@Ignore` annotation on the test recently. 🙈 
I'll bring it back